tests/framework/e2e/etcd_process.go

```diff
@@ -253,6 +259,34 @@ func (ep *EtcdServerProcess) Close() error {
 	return nil
 }
 
+func (ep *EtcdServerProcess) VerifySchemaVersion(lg *zap.Logger) error {
+	dbPath := datadir.ToBackendFileName(ep.cfg.DataDirPath)
+	be := backend.NewDefaultBackend(lg, dbPath)
+	defer be.Close()
+	ver, err := schema.UnsafeDetectSchemaVersion(lg, be.BatchTx())
+	if err != nil {
+		return err
+	}
+	maxStorageVersion := version.V3_6
+	if ep.cfg.ExecPath == BinPath.EtcdLastRelease {
+		maxStorageVersion = version.V3_5
+	}
```

Comment:
It might not be good to hard code the versions in test. I know it works for now because we haven't add version 3.7 yet. Have you considered making it more generic, e.g. get the version from the member right after starting the instance?

Reply:
done